plugins/org.eclipse.thym.android.core/src/org/eclipse/thym/android/core/adt/AndroidSDKManager.java

```diff
@@ -436,12 +436,12 @@ public void startAVDManager() throws CoreException{
 		ExternalProcessUtility processUtility = new ExternalProcessUtility();
 		StringBuilder command = new StringBuilder(getAndroidCommand());
 		command.append(" avd ");
-		processUtility.execAsync(command.toString(), null, null, null, null);
+		processUtility.execAsync(command.toString(), new File(toolsDir), null, null, null);
 	}
 	
 	private String getAndroidCommand(){
-		String command = addQuotes(toolsDir+"android");
-		return isWindows() ? "cmd /c "+ command : command;
+		String command = "android";
```

Comment:
Actually, Mac is my main machine but I made a mistake and used a wrong branch for testing. My bad. Anyway, base on my investigation to avoid the issue described in the bug we need to use a working directory. Do you have any concerns about such approach? For Mac and Linux we can use "./" to be sure that the correct android tool is used (even if another one is in PATH). Such problem does not occur Windows.